hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -789,4 +791,31 @@ public static String getOMAddressListPrintString(List<OMNodeDetails> omList) {
     printString.append("]");
     return printString.toString();
   }
+
+  public static String format(List<ServiceInfo> nodes, int port,
+                              String leaderId) {
+    StringBuilder sb = new StringBuilder();
+    int count = 0;
+    for (ServiceInfo info : nodes) {
```

Comment:
`nodes` may be in any order.  It would be nice if we could ensure ordering is consistent across all OMs.
<img width="1499" alt="Screenshot 2022-07-25 at 14 12 59" src="https://user-images.githubusercontent.com/6454655/180775101-4ea59eeb-ef87-48cc-a120-7dfb2b464021.png">
<img width="1503" alt="Screenshot 2022-07-25 at 14 13 06" src="https://user-images.githubusercontent.com/6454655/180775108-2a37f707-fc91-453f-9053-ab8a9484b0a6.png">